extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -87,8 +87,13 @@ amp-carousel.i-amphtml-carousel-has-controls .amp-carousel-button,
 
 amp-carousel .amp-carousel-button.amp-disabled {
   animation: none;
-  opacity: 0;
-  visibility: hidden;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
```

Comment:
I'm curious about this technique. Is this necessary for screen readers? IIUC this affects visual layout, but not interpretation by screen readers.

How is this different than, say, `display: none`?

Reply:
This technique is recommended to allow screen readers to find and focus on elements, while still having them 'visually hidden'. The technique is outlined [here](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/) that was mentioned in the original issue.

`display: none` will not allow screen readers to discover those elements when they should. Additionally, the screen reader will lose focus on the element then it "disappears" causing it to reset its focus (usually to the body of the doc) which is a jarring experience.

Reply:
I see, thanks for the explanation! Let's add a TODO to set this as a common pattern (like these props under a classname) so we're not pasting properties around.